samples/NovelRT.DotNet.Sample/CMakeLists.txt

```diff
@@ -0,0 +1,86 @@
+find_package(Dotnet ${NOVELRT_DOTNET_VERSION} REQUIRED)
+
+set(DOTNET_SAMPLE_CSPROJ ${CMAKE_CURRENT_LIST_DIR}/NovelRT.DotNet.Sample.csproj)
+
+set(DOTNET_SAMPLE_SOURCES
```

Comment:
This can just be `SOURCES`

Reply:
We weren't doing this elsewhere (see `ENGINE_SOURCES` for example) and that's why I mirrored it here.

Reply:
Yeah; those places are actually unnecessarily prefixed too, it's just that "if it ain't broke, don't fix it" applied :stuck_out_tongue:

Reply:
Happy to simplify, just to be clear, just wanted to call the above out since we should likely do it everywhere.

Reply:
Yeah, we should.